patterns/archive.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Title: Archive
```

Comment:
If we don't want this to hidden maybe it needs a better name and description.

Reply:
something List of posts in three columns?